kolibri/plugins/coach/assets/src/views/common/QuizLessonDetailsHeader.vue

```diff
@@ -20,6 +20,7 @@
             />
           </h1>
           <StatusElapsedTime
+            v-if="examOrLesson !== 'exam'"
```

Comment:
Oh the thing is that we use this `QuizLessonDetailsHeader` for quizzes and lessons. And we were updating the quiz page to have the creation time as part of the `QuizStatus` component, so in the figma spec we have that we dont need to show the creation time here in the header anymore.

But as this header component was also being used for lessons, and we dont have yet the creation time in the `LessonStatus` component (which I assumed will be part of another issue to update the lessons page), I preferred to put this conditional of `v-if="examOrLesson !== 'exam'"` so that it is not shown in the quizzes as it is in the figma spec, but not "accidentally" remove it from the lessons yet, and rather do it on purpose once the lessons page has been refactored.